app/src/main/java/com/theuhooi/uhooipicbook/modules/monsterlist/MonsterListRecyclerViewAdapter.kt

```diff
@@ -0,0 +1,59 @@
+package com.theuhooi.uhooipicbook.modules.monsterlist
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import coil.api.load
+import com.theuhooi.uhooipicbook.R
+import com.theuhooi.uhooipicbook.modules.monsterlist.MonsterListFragment.OnListFragmentInteractionListener
+import com.theuhooi.uhooipicbook.modules.monsterlist.entity.MonsterContent.MonsterItem
+import kotlinx.android.synthetic.main.item_monster_list.view.*
+
+class MonsterListRecyclerViewAdapter(
+    private val monsters: List<MonsterItem>,
+    private val listener: OnListFragmentInteractionListener?
+) : RecyclerView.Adapter<MonsterListRecyclerViewAdapter.ViewHolder>() {
+
+    // MARK: Stored Instance Properties
+
+    private val onClickListener: View.OnClickListener
```

Comment:
Kotlin properties can be initialized with constructor args:

```kotlin
private val onClickListener  = View.OnClickListener { v ->
    // do something
}
```

Reply:
It's possible with Swift😅
Fix it.